sp800-63b/sec5_authenticators.md

```diff
@@ -112,7 +112,7 @@ If the out of band authenticator sends an approval message over the secondary co
 
 Out of band verifiers SHALL generate a random authentication secret with at least 20 bits of entropy using an approved random number generator. They then optionally signal the device containing the subscriber's authenticator to indicate readiness to authenticate.
 
-Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP (or other software-based) service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
+Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP, chat, or other software-based service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
```

Comment:
I'm not clear on what you mean by "chat". If you're referring to services like iMessage that use telephone numbers for addresses, there isn't an association to a telephone number that can be looked up as one can to see if it is a mobile number. The sender just needs to send via SMS and not the chat service (although one could imagine a chat service that is bound to a specific device, and might therefore be acceptable).